recipes/ompl/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "ompl" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ompl/ompl/archive/{{ version }}.tar.gz
+  sha256: e782f723e1821f80ba6d6c61e65eed73eefd2b29c9c935f0e0b181d9395a0218
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkg-config
+  host:
+    - boost-cpp >=1.72
+    - eigen
+    - flann
+    - libode
+  run:
+    - libode
+    - boost-cpp >=1.72
```

Comment:
This is wrong as indicated in my previous comment.

Reply:
I was actually wondering wether run_exports on boost-cpp wouldn't automatically add boost-cpp as run dependency with the correct pinning?